skins/cat17/faq-page/src/components/Answer.vue

```diff
@@ -2,7 +2,8 @@
   <div id="answer" class="space-around">
   	<p>
 	{{ content }}
-	<span v-if="isPreview" class="link">some long german word here...</span>
+	<span v-show="!collapsed">The rest of the text appears here! Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</span>
+	<span v-if="isPreview" @click="collapsed = !collapsed;" class="link">some long german word here...</span>
```

Comment:
I think you might already account for the 2 different texts inside the link, depending on `collapsed` state. I've [asked in the ticket](https://phabricator.wikimedia.org/T202868#4644719) what those texts should be.

Reply:
Right, I've added `Expand` and `Collapse` for now.